bin/test-workflow/kubernetes/conjur-cli.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Do we still need all of the YAML manifests in the `kubernetes/` and `openshift/` subdirectories? I would think all of this would be replaced with the application deploy charts/subcharts.

The one YAML manifest that we might want to keep for the short term, I suppose, is this `conjur-cli.yml` manifest, although it would be good to migrate the Conjur CLI deployment to the parent Helm chart at some point.

Reply:
Removed unused yaml files